src/gui/setup.py

```diff
@@ -0,0 +1,31 @@
+import pygame
+
+from src import support
+from src.gui.interface import dialog, emotes
+from src.gui.interface.indicators import Indicators
+
+
+def setup_gui():
+    # <editor-fold desc="Dialog">
```

Comment:
What are these editor-fold comments about?

Reply:
I have found them more useful than a comment that spans an entire line to title a section of code. Instead, I opted for these `<editor-fold>` tags as they integrate well with my IDE (PyCharm).
Inspired by your comment, I just did a little research and found that a more widely used option for folding is `#region` comments, so I'll probably change it to that. Or remove it altogether, if you find them more confusing or distracting.

Reply:
I didn’t know that. I should try this